nixos/modules/virtualisation/kvmgt.nix

```diff
@@ -49,7 +49,9 @@ in {
     };
 
     boot.kernelModules = [ "kvmgt" ];
-    boot.kernelParams = [ "i915.enable_gvt=1" ];
+    boot.extraModprobeConfig = ''
+      options i915 enable_gvt=1
```

Comment:
At least this option is problematic. Many people load i915 already in initrd i.e. to be able to enter a password.
Since we don't have the configuration in initrd, it would be ignored in this case.

Reply:
Ah hang on. you also added it to the initrd... nevermind.

Reply:
The purpose of this PR is exactly to introduce these options to initrd - it works for me now.